packages/flame/lib/src/test_helpers/random_test.dart

```diff
@@ -0,0 +1,99 @@
+import 'dart:math';
+import 'package:meta/meta.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+final _rnd = Random(); // for generating seeds
+const _maxSeed = 4294967296;
+
+/// This function is equivalent to `test(name, body)`, except that it is
+/// better suited for randomized testing: it will create a Random
+/// generator and pass it to the test body, but also record the seed
+/// that was used for creating the random generator. Thus, if a test
+/// fails for a specific rare seed, it would be easy to reproduce this
+/// failure.
+///
+/// In order for this to work properly, all random number generation
+/// within `testRandom()` must be performed through the provided random
+/// generator.
+///
+/// Example of use:
+///
+///     testRandom('description', (Random random) {
+///       expect(random.nextDouble() == random.nextDouble(), false);
+///     });
```

Comment:
This should be in a code block using backticks instead. That is the standard we been handling in our documentation

Reply:
ty, good to know